### ISSUE TITLE
Fix recently translated Filebeat modules where integer coercions were added

### ISSUE THREAD

**Issue by webmat** [elastic/beats#9598]
Many of the recently Filebeat modules recently translated to ECS also added more integer coercion, for some of the fields that used to remain as strings (e.g. PID, port, etc).

This was done by adding `:int` in the grok expression, which may not be enough for all of these fields. We should review and use `:long` to perform the coercion where necessary.

Modules already merged, affected by this:

- [ ] #9245
- [ ] #9117
- [ ] #9084
- [ ] #9081
- [ ] #9138
- [ ] #9135

PRs already open, but not merged yet (should be fixed directly in their respective PRs):

- [ ] #8963
- [ ] #9293
- [ ] #9294
- [ ] #9297
- [ ] #9301
- [ ] #9308
- [ ] #9315
- [ ] #9005

This potential problem came up first here: https://github.com/elastic/beats/pull/8963#discussion_r242058280

**Comment by ruflin** [comment]
I suggest we move all of them to `long` as we also convert `type: int` in the fields.yml automatically to long.

**Comment by webmat** [comment]
Yes, I agree. I'm even doing that for HTTP status codes, which are very far from overflowing the capacity of a signed int. I'm going for the most straightforward approach possible: "Always use long, don't overthink it" 😂
